src/App.jsx

```diff
@@ -1,25 +1,37 @@
 // src/App.jsx
-
 import { Toaster } from "react-hot-toast";
-import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import { createBrowserRouter, Outlet, RouterProvider } from "react-router-dom";
 
 import Home from "./pages/Home";
 import NotFound from "./pages/NotFound";
-
 import "./styles/App.css";
+import Header from "./components/Header/Header";
+
+const Layout = () => (
+  <>
+    <Header />
+    <Outlet />
+  </>
+);
```

Comment:
You can move this in a separate Layout.tsx file inside components, and cut-paste the `main` from Home.jsx:
` <main className="container mx-auto px-6 py-12">`

That ways, Layout will remain reusable and kind of like entry-point

Reply:
I meant that you can move `Layout` component into a separate file itself